tests/test_diffraction_objects.py

```diff
@@ -284,23 +284,7 @@ def test_init_invalid_xtype():
             },
             {"xtype": "tth", "yarray": np.array([4.1, 5.1, 6.1, 7.1, 8.1, 9.1])},
         ),
-    ],
-)
-def test_scale_to(org_do_args, target_do_args, scale_inputs, expected):
-    original_do = DiffractionObject(**org_do_args)
-    target_do = DiffractionObject(**target_do_args)
-    scaled_do = original_do.scale_to(
-        target_do, q=scale_inputs["q"], tth=scale_inputs["tth"], d=scale_inputs["d"], offset=scale_inputs["offset"]
-    )
-    # Check the intensity data is the same as expected
-    assert np.allclose(scaled_do.on_xtype(expected["xtype"])[1], expected["yarray"])
-
-
-@pytest.mark.parametrize(
-    "org_do_args, target_do_args, scale_inputs",
-    [
-        # Test expected errors produced from scale_to() with invalid inputs
-        (  # C1: none of q, tth, d, provided, expect ValueError
+        (  # C5: none of q, tth, d, provided, expect to scale on the maximal x-arrays
```

Comment:
new test case

Reply:
put this one first?  I think it is worth the extra effort as this is our new default behavior.